{
    "checksum": "dee8f1df6633f991d1872849b20de1697a0f9a90f5e92469a7baec6c3f04d4aa",
    "file": "errors.hpp",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-c++"
    },
    "number_of_duplicates": 11,
    "package": "boost",
    "path": "boost/1.34.1-14/boost/python/errors.hpp",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 259566,
        "license": "/copyright/license/boost/1.34.1-14/",
        "metric": {
            "size": 116412
        },
        "pts_link": "https://tracker.debian.org/pkg/boost",
        "sloc": [
            [
                "cpp",
                642395
            ],
            [
                "xml",
                56450
            ],
            [
                "python",
                17612
            ],
            [
                "ansic",
                14520
            ],
            [
                "sh",
                2265
            ],
            [
                "yacc",
                858
            ],
            [
                "perl",
                481
            ],
            [
                "makefile",
                478
            ],
            [
                "lex",
                94
            ],
            [
                "sql",
                74
            ],
            [
                "csh",
                6
            ]
        ],
        "suites": [
            "lenny"
        ]
    },
    "raw_url": "/data/main/b/boost/1.34.1-14/boost/python/errors.hpp",
    "stat": {
        "perms": "rw-r--r--",
        "size": 1407,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.34.1-14"
}